README.md

```diff
@@ -15,14 +16,46 @@ jobs:
       uses: scientific-python/upload-nightly-action@82396a2ed4269ba06c6b2988bb4fd568ef3c3d6b # 0.6.1
       with:
         artifacts_path: dist
-        anaconda_nightly_upload_token: ${{secrets.UPLOAD_TOKEN}}
+        anaconda_nightly_upload_token: ${{secrets.ANACONDA_TOKEN}}
```

Comment:
Why is this getting changed?

Reply:
This is to suggest that since projects will use the same API key for their uploads and removals, having "UPLOAD" in the name (`UPLOAD_TOKEN`) wouldn't make sense for removals. Is there a better way to convey this information?

Reply:
Oh, your comment above of separating the actions into different sections should fix this, let me revert this change.